app/components/guides.css

```diff
@@ -1,4 +1,8 @@
 .guide {
-  composes: guide from 'css-modules-to-scoped-css/components/welcome-page-copy.css';
```

Comment:
The `composes` feature was used to create some sort of class inheritance. The imported class is added to the element along with the class rename. 

In other words, the template contains a `<a class="guide">`; in the built DOM, it results to `<a class="_guide_11ep6i _guide_f42zxn">`, the first class being the one defined in `guides.css`, and the second class being the one defined in `welcome-page-copy.css`.

In this example, `<Guides>` component is invoked by <WelcomePageCopy />, which creates an opportunity to demo the `{{scoped-class}}` helper from ember-scoped-css.